app/javascript/components/rooms/room/FeatureTabs.jsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import { Tabs, Tab, Placeholder } from 'react-bootstrap';
 import { useTranslation } from 'react-i18next';
-import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
 import Presentation from './presentation/Presentation';
 import RoomSettings from './room_settings/RoomSettings';
 import SharedAccess from './shared_access/SharedAccess';
 import { useAuth } from '../../../contexts/auth/AuthProvider';
 import useSiteSetting from '../../../hooks/queries/site_settings/useSiteSetting';
 import RoomRecordings from '../../recordings/room_recordings/RoomRecordings';
+import useRoom from '../../../hooks/queries/rooms/useRoom';
 
-export default function FeatureTabs({ shared }) {
+export default function FeatureTabs() {
   const { t } = useTranslation();
   const { isLoading, data: settings } = useSiteSetting(['PreuploadPresentation', 'ShareRooms']);
   const currentUser = useAuth();
 
+  const { friendlyId } = useParams();
+  const { data: room } = useRoom(friendlyId);
```

Comment:
This feels not optimised.
We are querying the Room inside every Feature tab - it could be passed down from the Room component.
Not sure how React Query works - maybe it is not too bad.
But it is either that or a (somewhat) major refactoring with placeholders etc.